docs/changes/utils/16025.bugfix.rst

```diff
@@ -0,0 +1,9 @@
+Fix rare signature incompatibilities between helper and helped array functions.
```

Comment:
This note is literally copy-pasted from #15710, only the list of functions is different. I don't know if that's undesirable somehow.